src/features/auth/components/ActivateAccountForm.tsx

```diff
@@ -15,23 +16,28 @@ type Props = {
   onSubmit: (data: FormData) => void;
 };
 
-const schema: yup.SchemaOf<FormData> = yup.object().shape({
-  password: yup
-    .string()
-    .required(Constants.errorMessages.NEW_PASSWORD_REQUIRED)
-    .min(8, Constants.errorMessages.PASSWORD_LENGTH)
-    .matches(Constants.patterns.LOWERCASE_REGEX, Constants.errorMessages.PASSWORD_LOWERCASE)
-    .matches(Constants.patterns.UPPERCASE_REGEX, Constants.errorMessages.PASSWORD_UPPERCASE)
-    .matches(Constants.patterns.SYMBOL_REGEX, Constants.errorMessages.PASSWORD_SPECIAL_CHARACTER)
-    .matches(Constants.patterns.DIGIT_REGEX, Constants.errorMessages.PASSWORD_NUMBER),
+export const ActivateAccountForm: FC<Props> = ({ onSubmit }) => {
+  const { t } = useTranslation(['translation', 'common']);
 
-  passwordConfirmation: yup
-    .string()
-    .required(Constants.errorMessages.CONFIRM_PASSWORD_REQUIRED)
-    .oneOf([yup.ref('password')], Constants.errorMessages.PASSWORD_MUST_MATCH),
-});
+  const schema: yup.SchemaOf<FormData> = yup.object().shape({
```

Comment:
The schemas need to be moved into the component so that they have access to `useTranslate()` and can update on re-render. I also refactored the `Constants.validationMessages` to make the code more intuitive and to support translations.